thing-url-adapter.js

```diff
@@ -408,10 +424,9 @@ class ThingURLDevice extends Device {
     }).then(() => {
       // Check for new events
       if (this.eventsUrl !== null) {
+        const headers = this.adapter.getHeaders(this.eventsUrl, true);
```

Comment:
You don't need the Content-Type header here, so you can omit the `true` argument.

Reply:
done